tdesign/src/Upload/Upload.tsx

```diff
@@ -178,6 +179,7 @@ const Upload = ({
           if (allSucceeded) {
             files.forEach((f) => {
               f.remove();
+              2;
```

Comment:
A mighty number, looks like :smile:

Reply:
lol this keeps happening because i use `cmd+2` to switch to the second pane - i even deleted one in this commit yesterday 😨